.changeset/large-stingrays-try.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-explore': minor
```

Comment:
Think this should be a `patch` release as the package is 0.x and this isn't a breaking change.